application.py

```diff
@@ -27,12 +28,21 @@
 def index():
     return "Project 1: TODO"
 
+@app.route("/admin")
+def admin():
+    users = db.query(User).all()
+    return render_template('users.html',users=users)  
+
 @app.route('/register', methods=['GET', 'POST']) 
 def register():
     if request.method == 'POST':
-         username = request.form['username']
-         password = request.form["password"]
-         return render_template('welcome.html',username=username)
+        import time   
+        username = request.form['username']
+        password = request.form["password"]
+        user = User(username=username, password=password, created_at=time.strftime('%Y-%m-%d %H:%M:%S'))
```

Comment:
How about setting the time as a default value of the database instead of setting it here.

Reply:
okay